frontend/src/components/Search/Search.jsx

```diff
@@ -28,14 +28,16 @@ class Search extends React.Component {
     static propTypes = {
```

Comment:
Note: This PR has been split into smaller commits for easier review.

This PR is best reviewed with whitespace turned off.